libs/expo/betterangels/src/lib/screens/AddEditClient/PersonalInfo/HmisProfiles.tsx

```diff
@@ -22,11 +22,13 @@ export default function HmisProfiles() {
     watch,
     control,
     formState: { errors },
+    resetField,
   } = useFormContext<UpdateClientProfileInput | CreateClientProfileInput>();
 
-  const { fields, append } = useFieldArray({
+  const { fields, append, remove } = useFieldArray({
     control,
     name: 'hmisProfiles',
+    keyName: 'keyId',
```

Comment:
not sure you saw this, but from their docs about `keyName`:
`this prop is no longer required and will be removed in the next major version.`

so ok for now, but FYI